core/sql/regress/hive/EXPECTED007

```diff
@@ -0,0 +1,370 @@
+.    .    1    hive_scan                       THIVE1                3.29E+001
+
+--- SQL operation complete.
+>>execute s;
+
+A            B          
+-----------  -----------
+
+          1            2
+
+--- 1 row(s) selected.
+>>
+>>get tables in view trafodion.sch.vhive1;
+
+Tables in View SCH.VHIVE1
+=========================
+
+HIVE.HIVE.THIVE1
+
+--- SQL operation complete.
+>>get tables in view trafodion.sch.vhive11;
+
+Tables in View SCH.VHIVE11
+==========================
+
+HIVE.HIVE.THIVE2
+
+--- SQL operation complete.
+>>
+>>get all tables in view trafodion.sch.vhive3;
+
+Tables in View SCH.VHIVE3
+=========================
+
+HIVE.HIVE.THIVE1
+HIVE.HIVE.THIVE2
+
+--- SQL operation complete.
+>>get views in view trafodion.sch.vhive3;
+
+Views in View SCH.VHIVE3
+========================
+
+TRAFODION.SCH.VHIVE2
+
+--- SQL operation complete.
+>>get views on table hive.hive.thive1;
+
+Views on Table _HV_HIVE_.THIVE1
+===============================
+
+SCH.VHIVE1
+SCH.VHIVE2
+
+--- SQL operation complete.
+>>get all views on table hive.hive.thive2;
+
+Views on Table _HV_HIVE_.THIVE2
+===============================
+
+SCH.VHIVE11
+SCH.VHIVE2
+SCH.VHIVE3
+SCH.VHIVEHBASE
+
+--- SQL operation complete.
+>>
+>>drop external table thive2 for hive.hive.thive2 cascade;
+
+--- SQL operation complete.
+>>showddl hive.hive.thive2;
+
+/* Hive DDL */
+CREATE TABLE THIVE2
+  (
+    A                                int
+  , B                                int
+  )
+  stored as textfile
+;
+
+--- SQL operation complete.
+>>showddl trafodion.sch.vhive3;
+
+*** ERROR[4082] Object TRAFODION.SCH.VHIVE3 does not exist or is inaccessible.
+
+--- SQL operation failed with errors.
+>>get all views on table hive.hive.thive2;
+
+--- SQL operation complete.
+>>
+>>obey TEST007(error_tests);
+>>
+>>create view trafodion.sch.vhive4 as select x.a, y.b from thive1 x, thive1 y;
+
+--- SQL operation complete.
+>>
+>>-- insert not allowed
+>>insert into trafodion.sch.vhive4 values (3,4);
+
+*** ERROR[4027] Table or view TRAFODION.SCH.VHIVE4 does not permit insertions.
+
+*** ERROR[8822] The statement was not prepared.
+
+>>
+>>-- del/upd not allowed
+>>delete from trafodion.sch.vhive1;
+
+*** ERROR[4223] Update/Delete on Hive table is not supported in this software version.
+
+*** ERROR[8822] The statement was not prepared.
+
+>>update trafodion.sch.vhive1 set b = 1;
+
+*** ERROR[4223] Update/Delete on Hive table is not supported in this software version.
+
+*** ERROR[8822] The statement was not prepared.
+
+>>
+>>--  traf view is not updatable
+>>delete from trafodion.sch.vhive4;
+
+*** ERROR[4028] Table or view TRAFODION.SCH.VHIVE4 is not updatable.
+
+*** ERROR[8822] The statement was not prepared.
+
+>>
+>>-- traf view must be in traf cat/sch
+>>create view vhive5 as select * from thive1;
+
+*** ERROR[3242] This statement is not supported. Reason: DDL operations can only be done on trafodion or external tables.
```

Comment:
Might be a little clearer to say that views can only be created in a trafodion schema. (The DDL operation is on two objects, thive1 and vhive5, but thive1 is not limited to being a trafodion or external table, and vhive5 doesn't exist yet.)